client/app/routers/AuthenticatedApp.tsx

```diff
@@ -244,6 +244,11 @@ const authenticatedRouter: Translated<RouteObject[]> = (t) =>
             },
           ],
         },
+        {
```

Comment:
Any reason you moved the sequence?

Reply:
With the current structure inside redux being modified (now experience points is not anymore coupled with users), and the Experience Points page being not only containing disbursement and forum but also the info regarding **all users**’ exp points, it’s only natural that the routes structure needs to be changed also (to get it out from users and only being a sub of course)